### PR TITLE
WIP Data science onramp data cleaning

### PR DESCRIPTION
**Pull request by TusharK54**
## Description
First PR for @bradmiro and @leahecole interns

Fixes #<ISSUE-NUMBER>

Note: It's a good idea to open an issue first for discussion.

## Checklist
- [ ] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.6` (see [Test Enviroment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] **Lint** pass:   `nox -s lint` (see [Test Enviroment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] Please **merge** this PR for me once it is approved.

### CODE REVIEW

File: data-science-onramp/data-cleaning/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+grpcio==1.29.0
+google-auth==1.16.0
+google-auth-httplib2==0.0.3
+google-cloud==0.34.0
+google-cloud-storage==1.28.1
+google-cloud-dataproc==0.8.0
```
Comment (busunkim96) [inline data-science-onramp/data-cleaning/requirements.txt:6]: These two packages should be the only dependencies you need to specify to get storage and dataproc.

`grpcio` and `google-auth` are pulled in as transitive dependencies, `google-auth-httplib2` is only needed for `google-api-python-client`, and `google-cloud` is a deprecated package.

File: data-science-onramp/data-cleaning/requirements-test.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+pytest==5.3.2
```
Comment (leahecole) [inline data-science-onramp/data-cleaning/requirements-test.txt:1]: Pytest also just got an update to 6.0.0

File: data-science-onramp/data-cleaning/clean_test.py [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+from google.cloud import dataproc_v1 as dataproc
+from google.cloud import storage
+import pytest
+
+# Set global variables
+PROJECT = os.environ["GCLOUD_PROJECT"]
+DATAPROC_CLUSTER = f"clean-test-{uuid.uuid4()}"
+BUCKET_NAME = f"clean-test-code-{uuid.uuid4()}"
+CLUSTER_REGION = "us-east4"
```
Comment (leahecole) [inline data-science-onramp/data-cleaning/clean_test.py:13]: Let's try changing this to us-central1, which has a higher CPU quota in our project than us-east4

File: data-science-onramp/data-cleaning/clean.sh [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Submit a PySpark job via the Cloud Dataproc Jobs API
+# Requires having PROJECT_ID, CLUSTER_NAME and BUCKET_NAME set as 
+# environment variables
+
+export CLUSTER_NAME=data-cleaning
+export PROJECT_ID=data-science-onramp
```
Comment (leahecole) [inline data-science-onramp/data-cleaning/clean.sh:6]: These variables do not line up with what the end user would be using

**Pull request closed without merging**